biz.aQute.bndlib/src/aQute/bnd/metatype/MetatypeVersion.java

```diff
@@ -20,4 +20,13 @@ public String getNamespace() {
 		return NAMESPACE_STEM + value;
 	}
 
+	static MetatypeVersion valueFor(String v) {
```

Comment:
AFAICT calling it valueOf won't compile as it conflicts with the built in valueOf method.  In any case I think it would be too confusing to use the same name.